src/main/java/seedu/address/MainApp.java

```diff
@@ -69,9 +69,9 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from {@code storage}'s talent connect and {@code userPrefs}. <br>
+     * The data from the sample talent connect will be used instead if {@code storage}'s talent connect is not found,
+     * or an empty talent connect will be used instead if errors occur when reading {@code storage}'s talent connect.
```

Comment:
I think "address book" here is used as a general noun, since it is not capitalized. 

Since TalentConnect is a address book, I think we can keep using that.

Also, I am not what does `reading storge's talent connect` mean.

Reply:
okay make sense, I will reverse this back to the original comment

Reply:
yeah I agree on this. TalentConnect is more just for the product name.